.idea/agent.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.idea stuff should not be added to the repo. I'm suggesting to update the root .gitignore with "gitignore" things that acceptance tests introduce.